sdcm/cassandra_harry_thread.py

```diff
@@ -0,0 +1,191 @@
+
+        return harry_summary, errors
+
+    def _run_stress_harry(self, node, loader_idx, stress_cmd, node_list):
+
+        CassandraHarryEvent.start(node=node, stress_cmd=stress_cmd).publish()
+        os.makedirs(node.logdir, exist_ok=True)
+
+        log_file_name = os.path.join(node.logdir, f'cassandra-harry-l{loader_idx}-{uuid.uuid4()}.log')
+        # Select first seed node to send the scylla-harry cmds
+        ip = node_list[0].private_ip_address
+
+        with CassandraHarryStressExporter(instance_name=node.ip_address,
+                                          metrics=nemesis_metrics_obj(),
+                                          stress_operation='write',
+                                          stress_log_filename=log_file_name,
+                                          loader_idx=loader_idx), \
+                CassandraHarryStressEventsPublisher(node=node, harry_log_filename=log_file_name):
+            result = None
+            try:
+                result = node.remoter.run(
+                    cmd=f"{stress_cmd} -node {ip}",
+                    timeout=self.timeout,
+                    log_file=log_file_name)
+            except Exception as exc:  # pylint: disable=broad-except
+                errors_str = format_stress_cmd_error(exc)
+                if "timeout" in errors_str:
+                    event_type = CassandraHarryEvent.timeout
+                elif self.stop_test_on_failure:
+                    event_type = CassandraHarryEvent.failure
+                else:
+                    event_type = CassandraHarryEvent.error
+                event_type(
+                    node=node,
+                    stress_cmd=stress_cmd,
+                    log_file_name=log_file_name,
+                    errors=[errors_str, ],
+                ).publish()
+            else:
+                CassandraHarryEvent.finish(node=node, stress_cmd=stress_cmd, log_file_name=log_file_name).publish()
+
+        return node, result
+
+    def run(self):
+        if self.round_robin:
+            loaders = [self.loader_set.get_loader()]
+        else:
+            loaders = self.loader_set.nodes if not self.use_single_loader else [self.loader_set.nodes[0]]
+        LOGGER.debug("Round-Robin through loaders, Selected loader is {} ".format(loaders))
+
+        self.max_workers = (os.cpu_count() or 1) * 5
+        LOGGER.debug("Starting %d cassandra-harry Worker threads", self.max_workers)
+        self.executor = concurrent.futures.ThreadPoolExecutor(max_workers=self.max_workers)
+
+        for loader_idx, loader in enumerate(loaders):
+            self.results_futures += [self.executor.submit(self._run_stress_harry,
+                                                          *(loader, loader_idx, self.stress_cmd, self.node_list))]
+            time.sleep(60)
+
+        return self
+
+    @staticmethod
+    def _parse_harry_summary(lines):  # pylint: disable=too-many-branches
```

Comment:
Please add in comment a sample of the output from the tool.
Is it only completed or failed?

Reply:
added, currently we only check if it's completed or not.
It doesn't output any summary of the write/read ops.